Core/include/Acts/Fitter/detail/VoidKalmanComponents.hpp

```diff
@@ -26,9 +26,9 @@ struct VoidKalmanComponents {
   ///
   /// @return void-calibrated measurement
   template <typename measurement_t, typename parameters_t>
-  Result<measurement_t> operator()(measurement_t m,
-                                   const parameters_t& /*pars*/) const {
-    return m;
+  Result<measurement_t> operator()(measurement_t measurement,
+                                   const parameters_t&) const {
+    return measurement;
```

Comment:
Can we keep these types of comments? First, I think they're helpful, second, I believe `clang-tidy` (which to be fair we're not running atm) uses these in some cases (in this particular case I don't think so though)